src/openstudio_lib/HVACSystemsController.cpp

```diff
@@ -331,22 +326,25 @@ void HVACSystemsController::update() {
     OSAppBase::instance()->processEvents();
 
     if (handle == REFRIGERATION) {
-      m_hvacSystemsView->hvacToolbarView->zoomInButton->setEnabled(false);
-      m_hvacSystemsView->hvacToolbarView->zoomOutButton->setEnabled(false);
+
+      m_hvacSystemsView->hvacToolbarView->controlsViewButton->hide();
 
       if (m_hvacSystemsView->hvacToolbarView->topologyViewButton->isChecked()) {
         m_refrigerationController = std::make_shared<RefrigerationController>();
 
         m_hvacSystemsView->mainViewSwitcher->setView(m_refrigerationController->refrigerationView());
-      } else if (m_hvacSystemsView->hvacToolbarView->gridViewButton->isChecked()) {
-        // TODO
 
-        m_hvacSystemsView->hvacToolbarView->zoomInButton->hide();
-        m_hvacSystemsView->hvacToolbarView->zoomOutButton->hide();
+        connect(m_hvacSystemsView->hvacToolbarView->zoomInButton, &QPushButton::clicked, m_refrigerationController->refrigerationView(),
```

Comment:
I'd be worried about the buttons being shown but not connected or connected to the wrong thing.  Would it be a good idea to start them out hidden and then you could see when they are shown the connections are right below?

Reply:
We own the code, and we're unlikely going to change this often. I'd say if it works, it works no?

I'm not opposed to the change, it's just some amount of work (maybe couple of hours with annoying manual testing) to change the logic from being show-default to hide-default. I'm not worried enough about maintainability, but if you really are, I support it.

Reply:
I'm really less familiar with this code so it's probably just my initial impression, no need to clean it up if its working